src/cbc_sdk/platform/alerts.py

```diff
@@ -921,6 +921,62 @@ def _query_implementation(cls, cb, **kwargs):
         return AlertSearchQuery(cls, cb).add_criteria("type", ["INTRUSION_DETECTION_SYSTEM"])
 
 
+class GroupedAlert(PlatformModel):
+    """Represents Grouped alerts."""
+    urlobject = "/api/alerts/v7/orgs/{0}/grouped_alerts"
+    swagger_meta_file = "platform/models/grouped_alert.yaml"
+
+    def __init__(self, cb, model_unique_id, initial_data=None):
+        """
+        Initialize the Grouped Alert object.
+
+        Args:
+            cb (BaseAPI): Reference to API object used to communicate with the server.
+            model_unique_id (str): ID of the alert represented.
+            initial_data (dict): Initial data used to populate the alert.
+        """
+        super(GroupedAlert, self).__init__(cb, model_unique_id, initial_data)
+        self._most_recent_alert = None
+
+        most_recent_alert = initial_data["most_recent_alert"]
+        if "type" in most_recent_alert:
```

Comment:
Needs to cater for type being something new that the SDK doesn't know about yet. It shouldn't break, should be a generic Alert.